tests/models/qwen2_vl/test_modeling_qwen2_vl.py

```diff
@@ -332,7 +333,7 @@ def setUp(self):
             }
         ]
         url = "https://qianwen-res.oss-accelerate-overseas.aliyuncs.com/Qwen2-VL/demo_small.jpg"
-        self.image = Image.open(requests.get(url, stream=True).raw)
+        cls.image = Image.open(requests.get(url, stream=True).raw)
```

Comment:
this is not good. It will setup the class even on CircleCI, although the tests are not really running as they are slow

Reply:
ah, my bad, for this, there is no model used in `setupClass`. I mixed several places in my brain .... sorry.